elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -72,12 +84,12 @@ async def verify_bugs_cli(runtime, verify_bug_status, output, bug_ids):
 
 class BugValidator:
 
-    def __init__(self, runtime: Runtime, output: str = 'text'):
+    def __init__(self, runtime: Runtime, use_jira: bool, output: str = 'text'):
         self.runtime = runtime
-        self.bz_data: Dict[str, Any] = runtime.gitdata.load_data(key='bugzilla').data
-        self.target_releases: List[str] = self.bz_data['target_release']
-        self.product: str = self.bz_data['product']
-        self.bug_tracker = bzutil.BugzillaBugTracker(self.bz_data)
+        self.config: Dict[str, Any] = JIRABugTracker.get_config(runtime) if use_jira else BugzillaBugTracker.get_config(runtime)
+        self.bug_tracker = JIRABugTracker(self.config) if use_jira else BugzillaBugTracker(self.config)
+        self.target_releases: List[str] = self.config['target_release']
+        self.product: str = self.config['product']
         self.bzapi = self.bug_tracker.client()
```

Comment:
Let's remove self.bzapi and directly use self.bug_tracker.client() in all places